gradle-193.properties

```diff
@@ -1,8 +1,8 @@
-ideaVersion=IU-193.4778-EAP-CANDIDATE-SNAPSHOT
-clionVersion=CL-193.4778-EAP-CANDIDATE-SNAPSHOT
+ideaVersion=IU-193.4932.9-EAP-SNAPSHOT
+clionVersion=CL-193.4932.12-EAP-SNAPSHOT
```

Comment:
Let's better use `EAP-CANDIDATE-SNAPSHOT` artifacts because sometimes there are some problems with sources code downloading for concrete EAP version